plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -400,7 +438,31 @@ protected List<CastTestCase> supportedCastTypePushdown()
                 new CastTestCase("c_decimal_10_2", "bigint", "c_bigint"),
                 new CastTestCase("c_decimal_19_2", "bigint", "c_bigint"),
                 new CastTestCase("c_decimal_30_2", "bigint", "c_bigint"),
-                new CastTestCase("c_decimal_negative", "bigint", "c_bigint"));
+                new CastTestCase("c_decimal_negative", "bigint", "c_bigint"),
+
+                new CastTestCase("c_char", "char(1)", "c_nchar"),
+                new CastTestCase("c_char_50", "char(10)", "c_char_10"),
+                new CastTestCase("c_char_50", "char(10)", "c_nchar_10"),
+                new CastTestCase("c_bpchar", "char(10)", "c_char_10"),
+                new CastTestCase("c_char_10", "char(50)", "c_char_50"),
+                new CastTestCase("c_varchar_15_unicode", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_nvarchar_15_unicode", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_char_10", "char(256)", "c_bpchar"),
+                new CastTestCase("c_char", "char(4096)", "c_char_4096"),
+
+                new CastTestCase("c_varchar_10", "varchar(10)", "c_varchar_10"),
+                new CastTestCase("c_varchar_50", "varchar(10)", "c_varchar_10"),
+                new CastTestCase("c_nvarchar_50", "varchar(10)", "c_nvarchar_10"),
+                new CastTestCase("c_text", "varchar(10)", "c_varchar_10"),
+                new CastTestCase("c_varchar_10", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_nvarchar_10", "varchar(50)", "c_varchar_50"),
+                new CastTestCase("c_varchar_10", "varchar(50)", "c_varchar_50"),
```

Comment:
`new CastTestCase("c_varchar_10", "varchar(50)", "c_varchar_50")` is duplicated.

Reply:
oops that was a miss. Sorry for leaving that behind :(